AwesomeBooks/modules/index.js

```diff
@@ -1,3 +1,4 @@
+import { DateTime } from 'luxon';
 import Book from './Book.js';
 
```

Comment:
- [ ] In this activity, we really want our code to follow ES6 standards, therefore kindly ensure refactor your functions to arrow functions , with regards to `display` method on _line 28_, `removeBtn` method on _line 55_, and `getDaySuffix` method on _line 89_.